novuhq/website
# feat: Blog integration with Strapi

**Describe what changes this pull request brings**
This pull request feat: adds `Blog` page and `detailed Blog` page + integration with `Strapi`.


**Steps to test**
1. Open [the page](https://websitemain13551-blogstrapiintegration.gtsb.io/blog)
2. Confirm that everything looks and works as expected
